include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -164,9 +166,39 @@ class BCX_API qrcode
             "The path to the configuration settings file."
         )
         (
+            "image,i",
+            value<bool>(&option_.image)->default_value(false)->zero_tokens(),
+            "Specifies that the QR code should be output in png format."
+        )
+        (
             "size,s",
             value<uint32_t>(&option_.size)->default_value(8),
-            "Specifies the module size in dots of the generated QR code."
+            "Specifies the module size in dots (pixles) of the generated QR code."
+        )
+        (
+            "margin,m",
+            value<uint32_t>(&option_.margin)->default_value(2),
+            "Specifies the margin size of the generated QR code."
+        )
+        (
+            "dpi,d",
+            value<uint32_t>(&option_.dpi)->default_value(72),
+            "Specifies the dpi of the generated QR code."
+        )
+        (
+            "version,v",
+            value<uint32_t>(&option_.version)->default_value(0),
+            "Specifies the version of the generated QR code."
+        )
+        (
+            "casesensitive,c",
```

Comment:
Use underscore if two words necessary, or preferably simplify to something like `casing`.